lib/pymedphys/_experimental/wlutz/interppoints.py

```diff
@@ -40,6 +40,7 @@ def transform_penumbra_points(points_at_origin, centre, rotation):
 
 
 def translate_and_rotate_transform(centre, rotation):
+    centre = np.array(centre, copy=False)
```

Comment:
docstring.
is centre being modified, or undnergoing type conversion?
This just looks weird.

Reply:
Here it is being type cast from "array like" to a "numpy array". The "copy=False" is so that if it is already an np.array essentially nothing will happen here.